ckanext/datawagovautheme/templates/snippets/facet_list.html

```diff
@@ -6,3 +6,38 @@ <h2 class="module-heading">
     {{ title }}
   </h2>
 {% endblock %}
+          {% block facet_list_items %}
+            {% with items = items or h.get_facet_items_dict(name) %}
+            {% if items %}
+              <nav>
+                <ul class="{{ nav_class or 'list-unstyled nav nav-simple nav-facet' }}">
+                  {% for item in items %}
+                    {% set href = h.remove_url_param(name, item.name, extras=extras, alternative_url=alternative_url) if item.active else h.add_url_param(new_params={name: item.name}, extras=extras, alternative_url=alternative_url) %}
+                    {% set label = label_function(item) if label_function else item.display_name %}
+                    {% set label_truncated = h.truncate(label, 22) if not label_function else label %}
```

Comment:
I think it might make more sense to remove `label_truncated` here and just include the full label in the ` <a href="{{ href }}" title="{{ label if label != label_truncated else '' }}">` block below.

Including the full label in the anchor's title seems like an obvious choice.